_summaries/bjorck2018understanding_1.md

```diff
@@ -1,19 +1,63 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{bjorck2018understanding}}
 # Please fill out info below
-author: # Your GitHub id
+author: # timchen0618
 score: # How did you like this paper 0(dislike) to 10(love)
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+## Background
+Normalizing the input data to zero-mean and constant standard deviation has long been recognized as good practice in training neural networks. 
+Batch normalization (BN) extend this idea to the input of each layer and achieve immense success in various areas. 
+It enables faster training, larger learning rates, and higher testing accuracy (and better generalizability). 
+However, the reason for the improvement is to be debated. 
+(e.g. the paper that propose BN said it was solving "internal covariate shift", but some suggested otherwise)
+
+## Core idea 
+The paper pointed out that BN is beneficial to training neural networks mainly because it allow for larger learning rates. 
+It also showed empirically that larger learning rates in networks without BN could lead to strong gradient updates, exploding activations and diverging loss . The same cannot be observed for neural networks with batch normalization, since the activation will be contained within a certain range, hence the stable loss and gradient. 
+
+## Experimental Results
+
+### Why is BN beneficial?
+As in the above figure, training with BN but with a small learning rate (lr=0.0001) does not improve testing accuracy. 
+Additionally, testing accuracy is higher when they chose large learning rate (0.1), displaying better generalization. 
+![Testing Accuracy](bjorck2018understandin_1a.png)
+
+
+### Divergence of NN Without BN
+Here they define 
+- relative loss (step-size) = new_loss/old_loss. 
+- divergence: when relative loss > $$10^3$$.
+At first few updates, network without BN exhibit great relative loss.
+![Step Size](bjorck2018understandin_1b.png)
+Activation of upper layers are extremely large, several orders of magnitude larger than lower ones. (Notice the scale)
+The authors claim that this implies divergence is caused by exploding activations.
+And BN fixes this problem by normalizing the input, preventing the large activations from propagating.
```

Comment:
I guess it is also worth to mention that means and variances of channels are quite constant on the log-scale as a function of depth of layers. and can also add a comment saying that normalizing the last layer might be the most important contribution of BN

Reply:
The first sentence I think is a good idea! I'll include that.

Reply:
For the second sentence, I think it also mentioned in the paper that "Figure 6 also shed light onto why the gradients of networks without BN tend to be so large in the final layers: the rows of the heatmap are highly correlated". In my understanding, BN solves it by distributing the gradients across classes, leading to the observation that final layers are normalized. Does it make sense?